### DISCUSSION TITLE
Icaros 3.3.3 Beta 1

### DISCUSSION THREAD

**Discussion by Xanashi** [Xanashi/Icaros#104] [Releases]
Good Afternoon Everyone,

I apologize once again for the long time periods between releases.
I've been dealing with a lot of IRL things, that has made it hard to find time to work on Icaros.

I still have some nice features like SVG and PDF thumbnailing in the works, but for today
you will have to settle with a slightly smaller release. 

As always for full transparency, here is what Icaros earned in donations last year:
**2023 - $471**

I can't thank you enough for all the donations, and wonderful feedback. 
And I always appreciate all the help with troubleshooting bugs and as well as the 
effort put in by the translators! ❤️ 

I hope you've all had a great 2024 so far, and I hope you all enjoy the new release!

<br/>

[![Icaros releases](https://img.shields.io/github/downloads/xanashi/icaros/v3.3.3b1/total?color=%232B6870&label=DOWNLOADS&style=for-the-badge)](https://github.com/Xanashi/Icaros/releases/tag/v3.3.3b1)

_______________________________________________________________________________________________________

### Icaros v.3.3.3 Beta 1

- Added support for VVC decoding
- Improved HEIF/AVIF thumbnailing support
- Fixed colors in EXR thumbnails (changed to sRGB color space)
- Fixed thumbnailing of PXZ files
- Fixed some unicode characters showing up wrong in properties
- Fixed rare MKV crash (Thank you Brett! ^__^)
- Fixed issue where Icaros failed to return properties of large files (4GB+) (Thank you dgonyier! ^__^)
- Fixed UNC Cache path fixes
- Updated GCC 
- Updated FFmpeg

<hr /><em>This discussion was created from the release <a href='https://github.com/Xanashi/Icaros/releases/tag/v3.3.3b1'>Icaros 3.3.3 Beta 1</a>.</em>

**Comment by Klaus1189** [comment]
Thank you very much ❤️❤️❤️

**Reply by Xanashi** [reply]
Appreciate it Klaus! ❤️

**Comment by Tarek-Hasan** [comment]
Hi, how are you?
Few days ago I realized, I didn't hear anything about next release for a long time. Nice to see you are still working on it.

**Reply by Xanashi** [reply]
Hi Tarak,

It's been a bumpy year for me IRL. 
But it's been really nice to put out a new release of Icaros for you all. ❤️
Hoping to work more on it this summer.

**Comment by FriedGenera** [comment]
Thank you for making this tool!
I was wondering if Github and shark007 are the places you officially post, or are places like Majorgeeks, Softpedia, LO4D etc also legitimate links for Icaros?

**Reply by Xanashi** [reply]
You're welcome ^__^
Github and the Shark007 forum is where I officially post, but places like Majogeeks, Videohelp, Softpedia are all legit places to download Icaros as well. There are also several legit package managers, like MS's winget, that serve Icaros, for users that like automation.

**Reply by Xanashi** [reply]
Also a million thanks to everyone that report updated version of Icaros to all these software sites and package managers! ❤️
It really means a lot to me!

**Comment by beachmobjellies** [comment]
Just found this project. Thank you for making this. This is brilliant

**Reply by Xanashi** [reply]
Thank you so much for the wonderful feedback, beachmobjellies. 
I hope you will continue to enjoy it for years to come!
